pkg/icingadb/ha.go

```diff
@@ -264,6 +264,9 @@ func (h *HA) controller() {
 
 // realize a HA cycle triggered by a heartbeat event.
 //
+// The given context MUST have a deadline, as the method will otherwise panic. This deadline is hardly enforced to
+// cancel the logic if it has expired.
```

Comment:
Hint: look up the meaning of "hardly", this sounds like it's not what you'd expect :sweat_smile:

Reply:
A hard working person or a hardly working person. There was a difference, I remember. Oops.